pkg/helm/helm_test.go

```diff
@@ -60,7 +61,7 @@ var (
 	password      = os.Getenv("DOCKER_PASSWORD")
 	insecure, _   = strconv.ParseBool(os.Getenv("DOCKER_INSECURE"))
 	chartRef      = ChartRef(hostname, namespace, chartName, tagName)
-	impl          = NewHelmerImpl(true)
+	impl          = NewHelmerImpl("")
```

Comment:
What is the purpose of defining TestLocalChartsMount if the path is empty?

Reply:
for this test the path is not empty: please see https://github.com/fybrik/fybrik/blob/995bd0d08d4f1490b6c96524fa1613f98ae66974/pkg/helm/helm_test.go#L142

Reply:
It would be cleaner to remove the global definition and use the right definition for each test.

Reply:
Sure, will do that. thanks